### PR TITLE
[16.0][MIG] sale_order_revision

### PR DESCRIPTION
**Pull request by CRogos**
based on v15 https://github.com/OCA/sale-workflow/pull/2381
required: https://github.com/OCA/server-ux/pull/599

### CODE REVIEW

File: sale_order_revision/view/sale_order.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            </field>
+        </field>
+    </record>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="priority">15</field>
+        <field name="arch" type="xml">
+            <xpath expr="//header" postion='inside'>
```
Comment (rven) [inline sale_order_revision/view/sale_order.xml]:
```suggestion
            <xpath expr="//header" position='inside'>
```

Comment (CRogos) [reply sale_order_revision/view/sale_order.xml]: I've added this in a separate commit for easier backport to v15

**Review by MohamedOsman7 — APPROVED** [review]
code + functional review LGTM

**Review by rven — APPROVED** [review]

File: sale_order_revision/view/sale_order.xml (hunk 1 above)
Comment (pedrobaeza) [reply sale_order_revision/view/sale_order.xml]: Although there's a typo, the default is position="inside", so this is not adding a real fix.

Comment (rven) [reply sale_order_revision/view/sale_order.xml]: @pedrobaeza I don't think it is working in v16 by default, we are already using this in our project, and the button was not showing at all, this change fixed the issue we had.

Comment (pedrobaeza) [reply sale_order_revision/view/sale_order.xml]: OK, maybe OWL is stricter in this way.

**Pull request merged**